packages/xrpl/test/wallet/index.ts

```diff
@@ -211,11 +246,11 @@ describe('Wallet', function () {
 
   describe('fromMnemonic', function () {
     const mnemonic =
-      'try milk link drift aware pass obtain again music stick pluck fold'
+      'assault rare scout seed design extend noble drink talk control guitar quote'
     const publicKey =
-      '0257B550BA2FDCCF0ADDA3DEB2A5411700F3ADFDCC7C68E1DCD1E2B63E6B0C63E6'
+      '035953FCD81D001CF634EB44A87940F3F98ADF2483D09C914BAED0539BE50F385D'
     const privateKey =
-      '008F942B6E229C0E9CEE47E7A94253DABB6A9855F4BA2D8A741FA31851A1D423C3'
+      '0013FC461CA5799F1357C8130AF703CBA7E9C28E072C6CA8F7DEF8601CDE98F394'
```

Comment:
Why were all these test cases changed? Were they no longer valid?

Reply:
Yeah, they were invalid bip39 encodings to begin with when I checked using this site and the bip39's `isValid` function: https://iancoleman.io/bip39/

Reply:
lmao